### PR TITLE
Add Interlingue translation

### PR DESCRIPTION
**Pull request by Carmina16**
Add a new Interlingue translation

### CODE REVIEW

File: po/ie.po [hunk 1]
```diff
@@ -0,0 +1,6155 @@
...
+msgstr "textu;redactor;"
+
+#: ../data/geany.glade.h:1
+msgid "_Toolbar Preferences"
+msgstr "Preferenties del ins_trumentarium"
+
+#: ../data/geany.glade.h:2
+#, fuzzy
+msgid "_Hide Toolbar"
+msgstr "Panel de instrumentarium"
```
Comment (codebrainz) [inline po/ie.po:44]: Should this be "__Ocultar__ panel de instrumentarium"?

File: po/ie.po [hunk 2]
```diff
@@ -0,0 +1,6155 @@
...
+msgstr "Inserter mu_ltilineari commenta"
+
+#: ../data/geany.glade.h:9
+msgid "_More"
+msgstr "_Plu"
+
+#: ../data/geany.glade.h:10
+#, fuzzy
+msgid "Insert File _Header"
+msgstr "Inserter un _file"
```
Comment (codebrainz) [inline po/ie.po:78]: Should this be "Inserter un __testa__ file"?

File: po/ie.po [hunk 3]
```diff
@@ -0,0 +1,6155 @@
...
+msgstr "ínvisibil"
+
+#: ../data/geany.glade.h:15
+msgid "_Insert \"include <...>\""
+msgstr "_Inserter «include <...>»"
+
+#: ../data/geany.glade.h:16 ../src/keybindings.c:513
+#, fuzzy
+msgid "Insert Alternative _White Space"
+msgstr "Inserter interspacie"
```
Comment (codebrainz) [inline po/ie.po:103]: Should this be "Inserter interspacie __alternative__"?

File: po/ie.po [hunk 4]
```diff
@@ -0,0 +1,6155 @@
...
+
+#: ../data/geany.glade.h:17
+msgid "_Search"
+msgstr "_Serchar"
+
+#: ../data/geany.glade.h:18
+#, fuzzy
+msgid "Open Selected F_ile"
+msgstr "Crear un f_ile"
```
Comment (codebrainz) [inline po/ie.po:112]: Should this be "__Aperte__ li file __selecte__" or similar?

**Comment by codebrainz** [comment]
Cool language, @Carmina16 do you know if this is widely used in any country/region as the normal/native language?

File: po/ie.po (hunk 4 above)
Comment (Carmina16) [reply po/ie.po:112]: Those are fuzzy (untranslated) strings.

Comment (codebrainz) [reply po/ie.po:112]: Yeah sorry I was just trying to help to defuzz by providing context. I know Geany well but not so much Interlingue :)

**Comment by Carmina16** [comment]
As far as I know no artificial language has a normative status anywhere.

**Pull request merged**